<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqExportToVtkJsDialog</class>
 <widget class="QDialog" name="pqExportToVtkJsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="selectSaveFileButton">
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="saveFileName">
     <property name="text">
      <string>file_name.vtkjs</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="embedInGlanceHtml">
     <property name="text">
      <string>Embed in ParaView Glance HTML?</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="writeTextures">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, the full-size texture will be embedded in the VTK.js file as a JPEG.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Embed textures in VTK.js file?</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="writeTextureLODs">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, texture LODs will be written as JPEG files in the same directory as the save file. The texture LODs are a series of decreasing resolution files which are intended to be uploaded to the web at the base URL. They will be downloaded sequentially when the VTK.js file is loaded.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Write texture LODs?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="paraviewGlanceFileName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>ParaViewGlance.html</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="paraviewGlanceFileNameLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>File name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="selectParaViewGlanceFileButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="textureLODsBaseUrlLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The base URL where the texture LODs will be available. It is the user's responsibility to upload the texture LODs to this URL.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Texture LOD Base URL:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="textureLODsBaseSizeLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The smallest resolution texture LOD file will be smaller in size than the base size. All other texture LODs will be larger. This base size indicates to the writer when the LOD file writing should stop.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Texture LOD Base Size:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="textureLODsBaseUrl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The base URL where the texture LODs will be available. It is the user's responsibility to upload the texture LODs to this URL.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QSpinBox" name="textureLODsBaseSize">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The smallest resolution texture LOD file will be smaller in size than the base size. All other texture LODs will be larger. This base size indicates to the writer when the LOD file writing should stop.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="suffix">
      <string> KB</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>saveFileName</tabstop>
  <tabstop>selectSaveFileButton</tabstop>
  <tabstop>writeTextures</tabstop>
  <tabstop>writeTextureLODs</tabstop>
  <tabstop>textureLODsBaseUrl</tabstop>
  <tabstop>textureLODsBaseSize</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pqExportToVtkJsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pqExportToVtkJsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeTextureLODs</sender>
   <signal>toggled(bool)</signal>
   <receiver>textureLODsBaseUrlLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeTextureLODs</sender>
   <signal>toggled(bool)</signal>
   <receiver>textureLODsBaseUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeTextureLODs</sender>
   <signal>toggled(bool)</signal>
   <receiver>textureLODsBaseSizeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeTextureLODs</sender>
   <signal>toggled(bool)</signal>
   <receiver>textureLODsBaseSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>embedInGlanceHtml</sender>
   <signal>toggled(bool)</signal>
   <receiver>paraviewGlanceFileNameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>embedInGlanceHtml</sender>
   <signal>toggled(bool)</signal>
   <receiver>paraviewGlanceFileName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>embedInGlanceHtml</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectParaViewGlanceFileButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
